core/commons/che-core-commons-tracing/src/main/java/org/eclipse/che/commons/tracing/TracingUtil.java

```diff
@@ -0,0 +1,22 @@
+package org.eclipse.che.commons.tracing;
+
+import io.opentracing.Span;
+
+public final class TracingUtil {
+
+  public static void setErrorStatus(Span span, Throwable e) {
+    TracingTags.ERROR.set(span, true);
+    TracingTags.ERROR_REASON.set(span, e.getMessage());
+    TracingTags.SAMPLING_PRIORITY.set(span, 1);
+  }
+
+  public static void setWorkspaceIdAndMachineName(
+      Span span, String workspaceId, String machineName) {
+    TracingTags.WORKSPACE_ID.set(span, workspaceId);
+    TracingTags.MACHINE_NAME.set(span, machineName);
+  }
+
+  public static void setWorkspaceId(Span span, String workspaceId) {
```

Comment:
Do you think we need this method? Does
```java
TracingUtil.setWorkspaceId(span, workspaceId);
```
looks better than
```
TracingTags.WORKSPACE_ID.set(span, workspaceId);
```
?

Reply:
+1 to remove it

Reply:
The idea when I initially wrote it was to have `setWorkspaceId` to match `setWorkspaceIdAndMachineId`. Since we'd be moving it to `TracingTags` I think it makes sense to not have either of these methods at all (and instead just use the `TracingTags` methods they wrap. 

Let me know what you think.

Reply:
I think it will be good enough to set tags via `TracingTags` without util methods